x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/execution/search/ScrollCursor.java

```diff
@@ -83,29 +90,46 @@ int limit() {
         return limit;
     }
     @Override
-    public void nextPage(Configuration cfg, Client client, NamedWriteableRegistry registry, ActionListener<RowSet> listener) {
+    public void nextPage(Configuration cfg, Client client, NamedWriteableRegistry registry, ActionListener<Page> listener) {
         log.trace("About to execute scroll query {}", scrollId);
```

Comment:
Shouldn't this log be conditioned by `log.isTraceEnabled()`?

Reply:
No because there's no object creation - the `isTrace` check makes sense when creating a string as a parameter so it's about whether it's worth the effort or not. In this case, the string is already in place.